src/Service/Ssm/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "async-aws/ssm",
+    "type": "library",
+    "description": "AsyncAws SSM client.",
+    "keywords": [
+        "aws",
+        "ssm"
+    ],
+    "license": "MIT",
+    "require": {
+        "php": "^7.2.5",
+        "ext-json": "*",
+        "async-aws/core": "^0.5"
+    },
+    "require-dev": {
+        "async-aws/core": "dev-master"
```

Comment:
I'm not sure this is the right way to get #492 ..

Reply:
No :)

It is easier to see the correct way if we had released 1.0. Of code 1.0
Was released, then the feature you added in #492 would make the next release 1.1.0. We should then update the core branch alias. 

With an updated beach alias, we should just require core ^1.1 here. It will automatically fetch the unreleased version until we actually release core 1.1.0. 

So... now when we are at 0.5.x... either we release a patch version of core now or we can use a branch alias. I think the patch version is simpler.